### ISSUE TITLE
Just remember previous selected area by default

### ISSUE THREAD

**Issue by romanr** [gifox/tracker#285]
After reading #283 that's closed issue, now I find out how to keep previous selection, and what good is it for me now? I need to **TIME TRAVEL INTO THE PAST** and select it there?

I would have never guessed that lock means "previous area". I actually clicked on it before wondering what that one is for, **while I was looking for menu to keep previous selection.** 

The lock symbol indicates security if it's a menu or button. Or in context or image selection indicates "lock aspect ratio."
There's no icon that would suggest "previous area". 

And why should it be?

This is must have functionality and should be enabled by default. Who don't want it to be previous area just in case? Who records everything on the first try? 
And if you don't want that area just click and drag! No extra effort required to reset. 
Why is it even a preference? and disabled by default? 

And if you must have it as preference, why can't you just show that option as text button?

For whom you're so insistent to keep that tool window as small as possible, for ants? 
Why you have to save every pixel of that window and show it as mysterious lock icon rather than human-friendly text button?

**Comment by iby** [comment]
Hey @romanr, I can feel so much passion in your feedback – thanks for stepping in!!! 😃

First, apologies for the poor experience – yes, UI/UX review is long overdue… I'll spare the details how we've ended up here, but I'm aware of the problems and I'm not happy with how things are either. Sequoia has reminded of how out of date the app is becoming and helped to kickstart this "revival" process: selection menu, interaction customizations, preferences, onboarding – all is on the list. The "tiny size problem" is the first item on the list – these improvements will start seeing light in the next release.

I really like how you've decomposed everything and showed the issue from different angles. Really appreciate you taking the time. I understand you're more upset with the app than happy, but would you mind if we get connected? First of all, I'd love to hear if you have any other thoughts – more than happy to do it publicly here, but I'm guessing the most issues are already reported. It would also be great to also hear a more "holistic" feedback.

I also feel like it might be great to bump some ideas off you in the future, if you're open to that – I'd love to connect, ping me on anywhere social or ${first}${last}@gmail.com.

**Comment by codyreppert** [comment]
I agree with this issue. I didn't even realize the lock icon did this so thankfully I can keep the functionality. I'm sure a lot of people would appreciate the lock button defaulting to on since it's not quite discoverable. 

Thank you for creating such an easy to use tool. I just purchased and am excited to continue to use it!

**Comment by timdmackey** [comment]
Hey there, I came on this thread because it was the first Google result when I searched for "gifox remember selection." I see now that it's described in the [documentation](https://gifox.app/docs/recording/#making-selection) as well!

I've been wondering for weeks why my new laptop didn't remember my previous selection the way my old computer does. I tried looking in the settings, but didn't find anything there. Like romanr, I didn't realize what the lock icon was supposed to do, or what "Open" meant. I think the terminology could be improved because I really don't understand what "Open" and "fixed" mean? To me the option is just "remember selection", yes or no.

Anyways, I'm glad to have found the solution now. I **_love_** this app and use it ever day, thank you iby!

**Comment by iby** [comment]
Howdy all! The long-overdue version 2.8.0 just landed with completely reworked selection / recording menu. Mac App Store version is still in review, will be available in a day or two.

Selection is remembered by default, and there's also a bunch of improvements and new features:

- Easier conversion between window ←→ area,
- Easier mouse / keyboard interaction control,
- Manual area resizing,
- Aspect ratio control,
- Pause and restart support.

<img width="1626" height="728" alt="Image" src="https://github.com/user-attachments/assets/f500a065-868e-4cd3-a191-9b682eb72859" />

There are still a few minor improvements on the list that are planned for the next couple of releases, main one being ability to drag the menu around when recording is in progress, and "destructive" actions confirmation.

But the current version should address the main issues with the outdated menu. Please give it a go, would love to hear any feedback or comments! 🙏

Again, thanks for everyone's patience and persistence – this has really helped to push this over the line.
